local-cli/generator-android/templates/src/app/BUCK

```diff
@@ -0,0 +1,68 @@
+import re
+
+# See [Docs](https://buckbuild.com/) to learn what BUCK is.
+# To run your application with BUCK you need:
+# - install BUCK
+# - `npm start` - to start the packager
+# - `cd android`
+# - `cp ~/.android/debug.keystore keystores/debug.keystore`
+# - `./gradlew :app:copyDownloadableDepsToLibs` - make all Gradle compile dependencies available to BUCK
+# - `buck install -r android-app` - compile, install and run application
+#
+
+lib_deps = []
+for jarfile in glob(['libs/*.jar']):
+  name = 'jars__' + re.sub(r'^.*/([^/]+)\.jar$', r'\1', jarfile)
```

Comment:
nit: Is there a way to use a simpler replacement than the regex? Might be a bit hard to read.